### DISCUSSION TITLE
Ubuntu Scale can't save after window zoom

### DISCUSSION THREAD

**Discussion by dejavuwl** [utmapp/UTM#3319] [General]
I changed scale in display settings to 200%,
after i zoom the window of UTM, the display turns black and then scale turned back to 100%

Why did this happen? Is this a Bug or just a feature of Ubuntu?

UTM 2.4.1(40)
macOS Monterey 12.0.1
VM OS: Ubuntu 20.04.3 arm aarch64
Machine: MBP 14 (M1 Pro)
<img width="1091" alt="截屏2021-11-18 11 01 00" src="https://user-images.githubusercontent.com/19871560/142344742-8199be23-0f23-44cb-9f01-9a073df84aaf.png">

**Comment by jose-heatly** [comment]
Did you manage to solve this issue?
